packages/create-webpack-app/bin/cli.js

```diff
@@ -0,0 +1,19 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import minimist from "minimist";
+import { Plop, run } from "plop";
+/* cSpell:disable */
```

Comment:
Does this disable cspell for the whole file or just the next line?

Reply:
section of code but I have found a better alternative
```suggestion
/* cSpell:words plopfile */
```